pages/platform/ai/notebook_guide_workspace/guide.en-gb.md

```diff
@@ -0,0 +1,64 @@
+---
+title: AI Notebooks - Workspace
+excerpt: Tutorial about how to understand AI Notebooks workspace
+updated: 2023-06-20
+---
+
+## Objective
+
+This guide details how AI Notebooks' workspaces work and best practices.
+
+## Instructions
+
+### What is a notebook's workspace?
+
+**AI Notebooks** are managed **Jupyter** or **Visual Studio Code** notebooks, linked to compute resources (CPUs, GPUs) and **storage**. 
+
+This **storage** includes both the remote containers you have attached to your notebook, and your notebook's default storage.
+
+Indeed, **every notebook has by default an internal object storage volume attached, fully managed by OVHCloud**, which is your workspace storage and is located at `/workspace`.
+
+### How do AI Notebooks' workspace work?
+
+When your **notebook is created**, a `/workspace` directory is automatically created. It **contains configuration information** such as **the machine learning framework** you have chosen to launch your notebook *(Miniconda, PyTorch, Tensorflow, ...)*, but also all **your installed python libraries**. This directory also allows you to store your data (datasets, codes, etc.).
```

Comment:
```suggestion
When your **notebook is created**, a `/workspace` directory is automatically created. It **contains configuration information** such as **the Machine Learning framework** you have chosen to launch your notebook *(Miniconda, PyTorch, TensorFlow, ...)*, but also all **your installed python libraries**. This directory also allows you to store your data (datasets, codes, etc.).
```

> *This directory also allows you to store your data (datasets, codes, etc.).*

> Not really... It's not advisable to do this, because the data and models cannot be used elsewhere. What's more, if the notebook is deleted, the user loses everything. It's better to have dedicated object container and plug in as a volume.